docker-compose.newnomis.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.1'
+services:
+  new-nomis-ui:
+    image: mojdigitalstudio/new-nomis-ui:latest
+    networks:
+      - hmpps
+    depends_on:
+      - oauth-server
+      - elite2-api
+    container_name: new-nomis-ui
+    ports:
+      - '3001:3001'
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:3001/health']
+    environment:
+      - PORT=3001
+      - NODE_ENV=dev
```

Comment:
convention dictated that this should either be `development` or `production`

Reply:
fixed this - think this was copied over from pathfinder.